### DISCUSSION TITLE
Add suggestion mode

### DISCUSSION THREAD

**Discussion by Lanhild** [growilabs/growi#8816] [Ideas / Requests]
Just like in Google Docs, a suggestion mode would be pretty useful.

![image](https://github.com/weseek/growi/assets/73610779/6cd0a771-0520-4c66-8be5-5470d89a4b8f)

**Comment by yuki-takei** [comment]
This request is related to the feature idea [**AI Inline Comments & Review #11096**](https://github.com/growilabs/growi/discussions/11096) proposed by the GROWI development team.

Please upvote and share your thoughts there!

**Reply by Lanhild** [reply]
My initial feature request is not a duplicate. My request was to have a suggestion mode, where a **human** can add in line comments
